test/check-application

```diff
@@ -22,46 +25,132 @@ registries = ['localhost:5000']
 registries = ['localhost:5000']
 """
 
+def checkImage(browser, name, owner):
+    browser.wait_present("#containers-images > section > table")
+    browser.wait_js_func("""(function (first, last) {
+        let items = ph_select("#containers-images > section > table tbody");
+        for (i = 0; i < items.length; i++)
+            if (items[i].innerText.trim().startsWith(first) && items[i].innerText.trim().endsWith(last))
+                return true;
+        return false;
+        })""", name, owner)
+
 class TestApplication(testlib.MachineCase):
 
     def setUp(self):
         # backup/restore pristine podman state, so that tests can run on existing testbeds
         # HACK: sometimes podman leaks mounts
         super().setUp()
         self.machine.execute("cp -a /var/lib/containers /var/lib/containers.orig")
+
+        # Create admin session
+        self.machine.execute("mkdir /home/admin/.ssh")
+        self.machine.execute("cp /root/.ssh/* /home/admin/.ssh")
+        self.machine.execute("chown -R admin:admin /home/admin/.ssh")
+        self.machine.execute("chmod 700 /home/admin/.ssh")
+        self.machine.execute("chmod 600 /home/admin/.ssh/*")
```

Comment:
you can also use a multi-line string and run through one .execute() command

How about `chmod -R go-wx /home/admin/.ssh` (but that's bikeshedding)